src/components/AddRemoveBtn.js

```diff
@@ -0,0 +1,58 @@
+import { useDispatch, useSelector } from 'react-redux';
+import React, { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { addBook, removeBook } from '../Redux/books/booksSlice';
+
+const AddBtn = () => {
+  const [title, setTitle] = useState('');
+  const [author, setAuthor] = useState('');
+  const dispatch = useDispatch();
+  const books = useSelector((state) => state.books.books);
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const newBook = {
+      id: uuidv4(),
+      title,
+      author,
+    };
+    dispatch(addBook(newBook));
+    setTitle(' ');
+    setAuthor(' ');
+  };
+
+  const handleRemove = (id) => {
+    dispatch(removeBook(id));
+  };
+
+  return (
+    <div>
+      <form onSubmit={handleSubmit}>
+        <input
+          type="text"
+          placeholder="Book title"
+          value={title}
+          onChange={(e) => setTitle(e.target.value)}
+        />
+        <input
+          type="text"
+          placeholder="Author"
+          value={author}
+          onChange={(e) => setAuthor(e.target.value)}
+        />
+        <button type="submit">
+          Add Book
+        </button>
+      </form>
+      <ul>
+        {books.map((book) => (
+          <li key={book.id}>
+            <button type="button" onClick={() => handleRemove(book.id)}>Remove</button>
+          </li>
+
+        ))}
+      </ul>
```

Comment:
- [ ] [Optional] [Highly recommended] The functionality of adding and removing a book is working as expected but the way the remove buttons are arranged under each other makes it hard to know which remove button belongs to which book. Please fix that.

![image](https://github.com/Surafels/Bookstore/assets/68796232/f47d4606-e49f-4547-a69f-7196fb4c35c3)